### DISCUSSION TITLE
How to install on Ruby on Rails

### DISCUSSION THREAD

**Discussion by kekeke29341** [WalletConnect/walletconnect-monorepo#726] [Developer Support]
Does anyone know how to install walletconnect into Ruby on Rails Apps?

https://stackoverflow.com/questions/68215866/how-to-add-js-plugin-via-yarn-to-rails-6-app
I have read these and tried.
However  not working well.

Please tell me how to do this

**Comment by lethetai2605** [comment]
yarn add @walletconnect/core
